### ISSUE TITLE
Edit File Metadata: Select All Files in Tab, Can't Edit Them All after clicking Edit Metadata

### ISSUE THREAD

**Issue by eaquigley** [IQSS/dataverse#2859]
Seen by @sbarbosadataverse and me today on production. 

We scrolled all the way through the files in the file tab, selecting all of them individually. We then clicked Edit Metadata and only 50 of the 66 files were available for editing. It seems like you cannot continuously scroll in the Primefaces component when on the edit page like you can on the view page.

**Comment by mheppler** [comment]
Related to **File Download: Changing the list of selected files for a multipage file list results in bad download all behavior. #3711**. Investigate as part of **Spike: Files DataTable, Selection, Paginator vs Scroll #4437**.

**Comment by mheppler** [comment]
Issue #4656, which removed the scrolling feature of the files table, was recently merged as part of [4.9.2](https://github.com/IQSS/dataverse/releases/tag/v4.9.2). I tried to replicate this issue on demo, with a dataset containing 59 files. I was able to individually select all 59 files, and when I clicked Edit Files > Metadata, the Edit Files pg displayed all 59 files.

Also, as part of #4656 a new dynamic counter and "Select all ## files in this dataset" text link was added to the top of the files table. This will help you know exactly how files have selected currently. Which comes in very handy as it pertains to the use case reported in this issue.

I suggest that we can close this issue. Will assign to QA and tag @kcondon to confirm and close.
